apps/opencs/model/world/refidcollection.hpp

```diff
@@ -44,7 +46,7 @@ namespace CSMWorld
 
         private:
 
-            const RefIdAdapter& findAdaptor (UniversalId::Type) const;
+            /*const*/ RefIdAdapter& findAdapter (UniversalId::Type) const;
```

Comment:
That is from one of sirherrbatka's commits.  I guess one of the adapters is modifying the document.  I'll see if there is a way around it.